microshift_release_notes/microshift-4-15-release-notes.adoc

```diff
@@ -269,4 +269,13 @@ Issued: 2024-04-08
 
 {product-title} release 4.15.8 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:1670[RHBA-2024:1670] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:1668[RHSA-2024:1668] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deploy_microshift-embed-in-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
```

Comment:
First two links generate 404 errors. I assume that this content will be available when this is release?

Reply:
Yes I think so.